common.gradle

```diff
@@ -1,10 +1,6 @@
 // Common definitions and constraints to be applied to the projects.
 // These are applied in the buildscript block.
-def useMavenLocal = false
 repositories {
-    if (useMavenLocal) {
```

Comment:
did this not work here?

Reply:
Same question...

Reply:
I removed it when it seemed that mavenLocal would be useless for testing the libraries. Now that I fixed the libraries to be mavenLocal-friendly, I'll put this back.